src/test/scala/com/cognite/spark/datasource/AssetsRelationTest.scala

```diff
@@ -91,16 +88,13 @@ class AssetsRelationTest extends FlatSpec with Matchers with SparkTest {
       rows => rows.length > 0)
     spark
       .sql(s"""
-        |select null as id,
-        |null as path,
-        |null as depth,
+        |select 1 as externalId,
         |'asset name' as name,
         |null as parentId,
         |'asset description' as description,
-        |null as types,
         |null as metadata,
         |'$assetsTestSource' as source,
-        |null as sourceId,
+        |10 as id,
```

Comment:
sourceId should become externalId, while the id above should remain "null as id", I think?